webrtc-stats.html

```diff
@@ -2218,6 +2220,38 @@ <h2>
                   formats have ways to vary the quantizer value within the frame.
                 </p>
               </dd>
+              <dt>
+                <dfn>psnrSum</dfn> of type <span class="idlMemberType">
+                record&lt;DOMString, double&gt;</span>
+              </dt>
+              <dd>
+                <p>
+                  MUST NOT [= map/exist =] for audio.
+                  The cumulative sum of the PSNR values of frames encoded by this sender.
+                  The record includes values for the "y", "u" and "v" components.
+                  The count of measurements is in {{psnrMeasurements}}.
+                </p>
+                <p>
+                  PSNR is defined in [[ISO-29170-1:2017]].
+                </p>
+              </dd>
+              <dt>
+                <dfn>psnrMeasurements</dfn> of type <span class="idlMemberType">unsigned long long
+                </span>
+              </dt>
+              <dd>
+                <p>
+                  MUST NOT [= map/exist =] for audio.
+                  The number of times PSNR was measured. The components of {{psnrSum}} are
+                  aggregated with this measurement.
+                </p>
+                <p>
+                  The PSNR is defined in [[ISO-29170-1:2017]].
+                </p>
+                <p class="note">
+                  The frequency of PSNR measurements is outside the scope of this specification.
```

Comment:
We should recommend that perceptually, Y-PSNR is used. And PSNR values can vary based on motion and encoder settings.

Typically, Average Y-PSNR is calculated by dividing with the number of frames.

Reply:
that division is done by the application.